index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tango Festival</title>
+
+    <!-- Styles -->
+    <link rel="stylesheet" href="./build/styles.css">
+
+    <!-- Fonts -->
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+
+    <!-- Google fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@900&display=swap" rel="stylesheet">
+
+
+    <!-- Google icons -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+
+    <!-- FA icons -->
+    <script src="https://kit.fontawesome.com/6ce384dd4f.js" crossorigin="anonymous"></script>
+
+  </head>
+  <body>
+    <div class="mobile-menu">
+      <i class="fa-solid fa-xmark"></i>
+    </div>
+    <div class="top-section desktop">
+      <img class="top-icon" src="./src/img/icons/fb-icon.svg" alt="facebook icon" height="10px" width="10px">
+      <img class="top-icon" src="./src/img/icons/twitter-icon.svg" alt="twitter icon" height="10px" width="10px">
+      <ul>
+        <li>English</li>
+        <li>My page</li>
+        <li>Logout</li>
+      </ul>
+    </div>
+    <nav class="nav-container">
+      <span class="material-symbols-rounded hamburger">menu</span>
+      <a href="./index.html">
+        <img class="logo" src="./src/img/project-logo/logo_tango_194x54.png" alt="international tango festivallogo">
+      </a>
+      <div class="right-side"></div>
+      <div class="right-side">
+        <ul class="desktop-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#program">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+      </div>
+    </nav>
+    <section id="header">
+      <div>
+        <p class="blockquote">"Put on your dancing shoes!!!"</p>
+        <h1 class="header-class">International <br> <span>Tango</span> <br> Festival</h1>
+      </div>
+      <div class="introduction">
+        <p><span class="quote">"The most important Tango event in South America"</span></p>
+        <p>The Internatiuonal Tango Festival will be held in Bogota, Colombia. The most famous milongueros from Colombia will present in this magnificent event.</p>
+      </div>
+      <div class="place">
+        <p class="date">2022.11.01 ~ 06 (monday to sunday)</p>
+        <p class="salon">@ Hotel Tequendama, Bogota , Colombia</p>
+      </div>
+    </section>
```

Comment:
- This section of your website doesn't match the design in the given template. While it is good to personalize the contents of your site, please remember that you are required to stick to the general design layout (background image, text color, font size, font weight, margins, borders e.t.c) as seen in the design template. Also, remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

# Your project(Please fix as it may apply for the about page) 👇:
-- There is a background image in the design template that is missing in your project 👍 
--Please stick to the correct font-color, font-size and font-weight for the texts in this section 🙏 
-- The bottom part of this section is overflowing into the next section. Please fix this 👍 
<img width="944" alt="tango home header" src="https://user-images.githubusercontent.com/73607512/197239242-171eb668-9e5d-4c24-9430-1244ad02f10f.PNG">